tests/test_browser.py

```diff
@@ -3,7 +3,7 @@
 import tempfile
 
 import pytest
-import setpath  # noqa:F401, must come before 'import mechanicalsoup'
+# import setpath  # noqa:F401, must come before 'import mechanicalsoup'
```

Comment:
The `setpath` import needs to be retained in all the test files. Maybe there's a better way to set the python path for testing these days, but I'm not sure what what would be.